compiler/rustc_session/src/config.rs

```diff
@@ -1400,9 +1400,8 @@ The default is {DEFAULT_EDITION} and the latest stable edition is {LATEST_STABLE
 pub fn rustc_short_optgroups() -> Vec<RustcOptGroup> {
     vec![
         opt::flag_s("h", "help", "Display this message"),
-        opt::multi_s("", "cfg", "Specify the features defined in the manifest
-                             and compile the relevant components based on
-                             cfg attributes.", "SPEC"),
+        opt::multi_s("", "cfg", "Configure the compilation environment.
+                             SPEC supports the syntax `NAME[=\"VALUE\"]`.", "SPEC"),
```

Comment:
I see all the tests with `compile-flats:--cfg` and they belong to the above forms.
Maybe we can just use `"NAME[=VALUE]"` in the `"SPEC"` position.

Reply:
so I just leave `NAME="VALUE"`? because 
> For examples, --cfg 'verbose' or --cfg 'feature="serde"'. These correspond to #[cfg(verbose)] and #[cfg(feature = "serde")] respectively.
https://doc.rust-lang.org/rustc/command-line-arguments.html#--cfg-configure-the-compilation-environment

Reply:
I'm a rust rookie. thanks for your help.